src/components/Jobs/JobDetail.tsx

```diff
@@ -1,4 +1,73 @@
-function JobDetail() {
-  return <div className="flex h-full flex-col" />;
+import { Button, Container, Loader, Tabs } from '@mantine/core';
+import { useParams } from 'react-router-dom';
+
+import JobDescriptionTab from '@/components/Jobs/JobDescriptionTab';
+import JobOverviewTab from '@/components/Jobs/JobOverviewTab';
+import { useAuth } from '@/hooks/useAuth';
+import { useJob, useKillJobMutation } from '@/hooks/useJobs';
+
+export default function JobDetails() {
+  const { jobId = '', region = '', env = '' } = useParams();
+  const { data: jobData } = useJob(env, region, jobId);
+  const { mutate } = useKillJobMutation();
+  const { user } = useAuth();
+
+  const tabs: string[] = ['Overview', 'Job Description', 'Stage Details', 'Job Status Output'];
```

Comment:
Don't need to explicitly type here.

Reply:
Where should this go? As these are constants and part of the component I thought this would be fine.

Reply:
No I mean you dont need the :string[] part so instead 

const tabs = ['Overview', 'Job Description', 'Stage Details', 'Job Status Output'];